### DISCUSSION TITLE
Add ability to assign a Prefix to a Region

### DISCUSSION THREAD

**Discussion by CLEEDEN** [netbox-community/netbox#12866] [Ideas]
I need to assign a Prefix and GPS coordinates to a Region

**Comment by candlerb** [comment]
Have a look at [Custom Fields](https://docs.netbox.dev/en/stable/customization/custom-fields/), especially Object fields.

You could add a custom field on Prefix pointing to its Region. Or you could have a custom multi-object field on Region, containing one or more Prefixes.

Adding GPS coordinates to a Region could be done with custom fields for Lat and Long. (However, since a Region contains multiple Sites, and possibly child Regions, it wouldn't normally be meaningful to pinpoint a Region to an exact GPS coordinate)
